src/modelfree/training/logger.py

```diff
@@ -79,6 +79,7 @@ def setup_logger(out_dir='results', exp_name='test', output_formats=None):
     timestamp = utils.make_timestamp()
     exp_name = exp_name.replace('/', '_')  # environment names can contain /'s
     out_dir = osp.join(out_dir, '{}-{}'.format(timestamp, exp_name))
+    out_dir = out_dir[0:255]
```

Comment:
Risky: what if `out_dir` is >255 characters before the join? We end up writing to wrong directory.

Linux has path limit of 4096 characters, filename limit is 255; so let's truncate filename instead.